src/main/java/org/parabot/core/ui/components/VerboseLoader.java

```diff
@@ -61,7 +61,7 @@ private VerboseLoader(String username, String password) {
         setOpaque(false);
 
         if (username != null && password != null) {
-            if (Core.inDebugMode() || manager.login(username, password, false)) {
```

Comment:
Previously, in debug mode, `manager.login()` is skipped and therefore credentials are not stored. This is the cause of the NPE thrown when we try to fetch BDN information, since credentials to authentication the connection are null.